### DISCUSSION TITLE
make ( hell board )for government people

### DISCUSSION THREAD

**Discussion by parisa4360** [WeAreMahsaAmini/Core#178] [Ideas]
https://twitter.com/Dr_Tohid_Azizi/status/1574666059365359617?s=20&t=ntitfpe6XVPPnkYRJnxu-A

we need to stop their voice also

**Comment by kaveh6** [comment]
some thing similar here : https://github.com/WeAreMahsaAmini/Core/discussions/47

**Comment by kaveh6** [comment]
please label this idea as duplicate as well
